samples/perf_hgemm.cpp

```diff
@@ -667,11 +694,30 @@ ROCWMMA_HOST void gemm_test(uint32_t m, uint32_t n, uint32_t k, float32_t alpha,
     auto macroTileSize
         = rocwmma::make_coord2d(TBLOCK_X / warpSize * WARP_TILE_X, TBLOCK_Y * WARP_TILE_Y);
 
+    // Device check for supported block and wave sizes
+    if(isGfx11() && (ROCWMMA_M > 16 || ROCWMMA_N > 16))
```

Comment:
Only block size supported is 16 x 16 for gfx11. Can possibly adjust to check
isGFX11() && ((ROCWMMA_M != ROCWMMA_N) || (ROCWMMA_M != 16))

Can do a similar check for gfx9 (16x16 and 32x32):
isGfx9() && ((ROCWMMA_M != ROCWMMA_N) || (ROCWMMA_M != 16 && ROCWMMA_M != 32))